db/migrate/20200114095428_create_whitehall_migration_timeline_entries.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class CreateWhitehallMigrationTimelineEntries < ActiveRecord::Migration[6.0]
+  def change
+    create_table :whitehall_migration_timeline_entries do |t|
+      t.string "entry_type"
```

Comment:
should this be null false?

Reply:
Yes, it should.  Oversight not including it.